assets/queries/cloudFormation/cloud_trail_multi_region_disabled/query.rego

```diff
@@ -0,0 +1,19 @@
+package Cx
+
+CxPolicy [ result ] {
+  resource := input.document[i].Resources[name]
+  resource.Type == "AWS::CloudTrail::Trail"
+  not checkRegion(resource)
+
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("Resources.%s.Properties", [name]),
+                "issueType":		"IncorrectValue", 
+                "keyExpectedValue": sprintf("'Resources.%s.Properties.IsMultiRegionTrail' is true", [name]),
+                "keyActualValue": 	sprintf("'Resources.%s.Properties.IsMultiRegionTrail' is false", [name])
+              }
+}
```

Comment:
Query should be divided into two. This is correct, however, if you divide into one query that checks if 'isMultiRegionTrail' is undefined and another that checks, when defined, if it is set to false. This way, you can be more specific in the type of issue, because when 'undefined' it should be 'MissingAttribute' and when 'false' it should be 'IncorrectValue'. Additionally, with the 'IncorrectValue' query I mentioned, the searchKey should be 'sprintf("Resources.%s.Properties.IsMultiRegionTrail", [name])'.